container-host-files/etc/scf/config/role-manifest.yml

```diff
@@ -906,13 +958,93 @@ instance_groups:
                   topologyKey: "beta.kubernetes.io/os"
   - name: statsd_injector
     release: statsd-injector
+  - name: bpm
+    release: bpm
+    properties:
+      bosh_containerization:
+        run:
+          capabilities: [ALL]
+        pod-security-policy: privileged
+- name: log-cache-scheduler
+  scripts:
+  - scripts/forward_logfiles.sh
+  - scripts/patches/fix_monit_rsyslog.sh
+  - scripts/patches/fix_logcache_certs_forwarder.sh
+  - scripts/patches/fix_logcache_certs_scheduler.sh
+  jobs:
+  - name: global-properties # needs to be first so images use it for processing monit templates
+    release: scf-helper
+  - name: log-cache-scheduler-properties
+    release: scf
+  - name: authorize-internal-ca
+    release: scf
+  - name: log-cache-scheduler
+    release: log-cache
+    # defaults
+  - name: log-cache-expvar-forwarder
+    release: log-cache
+    # defaults
+    properties:
+      bosh_containerization:
+        colocated_containers:
+        - loggregator-agent
+        run:
+          service-account: privileged
```

Comment:
Why does this need to be privileged? (There's probably a good reason, but we should document it)

Reply:
Will recheck.

Reply:
IIRC without it the fact that the loggregator-agent became privileged was not properly accounted for in the output of fissile. And I do not remember in full why the privileged on the BPM in this role was not enough.

You might wish to try to undo this and see if my recollection is true.